### PR TITLE
Do not use develop install on CI

### PR DESCRIPTION
**Pull request by vidartf**
Develop install can hide issues with setup scripts, as seen in #283.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,16 +27,12 @@ before_install:
       npm install -g codecov
     fi
 install:
-  - pip install --upgrade -e ".[test]"
-  - |
-    if [[ $GROUP == js ]]; then
-      cd nbdime-web
-      npm install
```
Comment (vidartf) [inline .travis.yml]: Note: The removal of the `npm install` line should also reveal if that step was not completed during `pip install`.

**Comment by vidartf** [comment]
Hmm... The argument for keeping the develop install is that we can use the cache for node modules. How much do we like quick CI? 😅

File: .travis.yml (hunk 1 above)
Comment (vidartf) [inline .travis.yml]: PS: This should actually only be removed if we keep the develop install, otherwise we need it.

**Comment by minrk** [comment]
Another option is to do a wheel install:

```
python setup.py bdist_wheel
pip install dist/*.whl
```

I don't think the `node_modules` cache is as important as the npm download cache.

**Comment by vidartf** [comment]
Green now :)

**Pull request merged**